src/reanimated2/commonTypes.ts

```diff
@@ -111,7 +111,11 @@ export interface AdapterWorkletFunction extends WorkletFunction {
   (value: NestedObject<string | number | AnimationObject>): void;
 }
 
-export type AnimatableValue = number | string | Array<number>;
+type AnimatableSingeValue = number | string | Array<number>;
```

Comment:
This is not the best name fot this type. Array doesn't seem to be a single value :/ Could we change this name? 🙏